javascript-modal/styles.css

```diff
@@ -0,0 +1,51 @@
+.open {
+  background-color: #4a4ae9;
+  color: #fff;
+  height: 3rem;
+  width: 13rem;
+  font-size: 25px;
+  border-radius: 6px;
+  border: none;
+}
+
+.hidden {
+  display: none;
+}
+
+.pop-up {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  width: 30rem;
+  height: 14rem;
+  margin-left: 35rem;
```

Comment:
That's a **_lot_** of margin 👀 

You should make `.modal` a flex container with `justify-content:center`